subs/pantry/src/Pantry/Types.hs

```diff
@@ -2269,7 +2270,13 @@ getGlobalHintsFile = do
 
 -- | Creates BlobKey for an input ByteString
 --
--- @sinc 0.1.0.0
+-- @since 0.1.0.0
 bsToBlobKey :: ByteString -> BlobKey
 bsToBlobKey bs =
     BlobKey (SHA256.hashBytes bs) (FileSize (fromIntegral (B.length bs)))
+
+-- | Identifier from a 'PackageLocationImmutable'
+pliIdent :: PackageLocationImmutable -> PackageIdentifier
+pliIdent (PLIHackage ident _ _) = ident
+pliIdent (PLIArchive _ pm) = pmIdent pm
+pliIdent (PLIRepo _ pm) = pmIdent pm
```

Comment:
@snoyberg there's already `packageLocationIdent`  -https://github.com/commercialhaskell/stack/blob/38826df905752a9c708c085f021c3c3b6d592879/subs/pantry/src/Pantry.hs#L1337

Reply:
Doh, thanks. I'll send a PR to fix that.